modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -464,7 +464,7 @@ endif::vsphere[]
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-dns-records_{context}"]
 === DNS records
-You must create DNS records for two static IP addresses in the appropriate DNS server for the vCenter instance that hosts your {product-title} cluster. In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster base domain that you specify when you install the cluster. A complete DNS record takes the form: `<component>.<cluster_name>.<base_domain>.`.
+You must create DNS records for certain static IP addresses in the appropriate DNS server for the vCenter instance that hosts your {product-title} cluster. In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster base domain that you specify when you install the cluster. A complete DNS record takes the form: `<component>.<cluster_name>.<base_domain>.`.
```

Comment:
Hello @sgaoshang and @jcpowermac . I swapped a numeric value ("two" or "three") for "certain" so to future proof this entry sentence in the event a DNS record is removed or added.